src/UI/Implementation/Component/Symbol/Icon/Standard.php

```diff
@@ -169,10 +167,24 @@ public function isOutlined() : bool
      * @param bool $is_outlined
      * @return Icon
      */
-    public function withIsOutlined(bool $is_outlined) : Icon
+    public function withIsOutlined(bool $is_outlined) : C\Symbol\Icon\Standard
     {
         $clone = clone $this;
         $clone->is_outlined = $is_outlined;
         return $clone;
     }
+
+    public function getIconPath() : string
+    {
+        $name = $this->getName();
+        if (!in_array($name, $this->getAllStandardHandles())) {
+            $name = self::DEFAULT_ICON_NAME;
+        }
+
+        $path = self::ICON_PATH;
+        if ($this->isOutlined()) {
+            $path = self::ICON_OUTLINED_PATH;
+        }
+        return $path . '/' . sprintf(self::ICON_NAME_PATTERN, $name);
+    }
```

Comment:
As with the according constants, I think this is a concern of the renderer not the component itself.

Reply:
you are so right, thanks ;)